build.gradle

```diff
@@ -262,13 +263,34 @@ task runBeamDependencyCheck() {
   dependsOn ":sdks:python:dependencyUpdates"
 }
 
+repositories {
+	
+	mavenCentral()
+}
+    // to use:
+    // dependencies {
+    //   compile library.java.slf4j_api
+dependencies {
+	
+	// https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on
+compile group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.64'
+
+// https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-client
+compile group: 'org.apache.hadoop', name: 'hadoop-client', version: '3.2.0'
```

Comment:
If you want to upgrade hadoop, you want to edit [BeamModulePlugin.groovy](https://github.com/apache/beam/blob/980e655a32a65f464ace85ff518b5d5b346677ba/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L376) since that is where we define shared dependency versions.